src/components/projects/CreateTaskModal.tsx

```
@@ -1,328 +1,95 @@
+
+type TaskData = Omit<Task, 'id' | 'projectId' | 'createdAt' | 'updatedAt'>;
 
 interface CreateTaskModalProps {
   projectId: string;
   onClose: () => void;
-  onTaskCreated?: () => void;
+  onCreate: (taskData: TaskData) => void;
 }
 
-export const CreateTaskModal: React.FC<CreateTaskModalProps> = ({ projectId, onClose, onTaskCreated }) => {
+export const CreateTaskModal: React.FC<CreateTaskModalProps> = ({ projectId, onClose, onCreate }) => {
```

ESLint

[error] 'projectId' is defined but never used.

(@typescript-eslint/no-unused-vars)